doc/source/whatsnew/v0.20.0.txt

```diff
@@ -624,7 +624,8 @@ Bug Fixes
 - Bug in ``pd.concat()`` in which concatting with an empty dataframe with ``join='inner'`` was being improperly handled (:issue:`15328`)
 
 
-
+- Bug in ``groupby.transform`` where calculating a ``timedelta`` from a ``datetime`` caused a ``ValueError`` (:issue:`15429`)
```

Comment:
just do a single note with the issue number. make this simpler, something like: .transform incorrectly changing the dtype of the output.